core/crypto/rand_darwin.odin

```diff
@@ -0,0 +1,11 @@
+package crypto
+
+import "core:fmt"
+import "core:sys/darwin"
+
+_rand_bytes :: proc(dst: []byte) {
+	res := darwin.SecRandomCopyBytes(count=len(dst), bytes=raw_data(dst))
+	if res != .Success {
+		panic(fmt.tprintf("crypto/rand_bytes: SecRandomCopyBytes returned non-zero result: %v", res))
```

Comment:
I wanted to bind to https://developer.apple.com/documentation/security/1394686-seccopyerrormessagestring to get a string message for the error but this returns a Foundation string, which we currently have bindings for it but those are in vendor, so can't use them